test/test_helper.rb

```diff
@@ -191,6 +199,13 @@ def sp_sql
         RETURN(420) |
     end
 
+    def sp_error_sql
+      %|CREATE PROCEDURE tinytds_TestPrintWithError
+        AS
+        PRINT 'hello'
+        RAISERROR('Error following print', 16, 1)|
```

Comment:
This test is specific to "message then error". We might want another test which is "message followed by more messages".

Maybe like:

```ruby
def sp_several_prints_sql
  %|CREATE or alter PROCEDURE tinytds_TestSeveralPrints
    AS
    PRINT 'hello 1'
    PRINT 'hello 2'
    PRINT 'hello 3' |
end

# ---

it 'must raise an error for each of a series of `print` messages' do
  messages.clear
  action = lambda { @client.execute("EXEC tinytds_TestSeveralPrints").do }
  assert_raise_tinytds_error(action) do |e|
    assert_equal ['hello 1', 'hello 2', 'hello 3'], messages.map { |e| e.message }, 'message list'
  end
end
```